src/patternfly/components/Accordion/accordion-expanded-content.hbs

```diff
@@ -1,5 +1,13 @@
 <{{#if accordion--IsDefinitionList}}dd{{else}}div{{/if}} class="pf-c-accordion__expanded-content{{#if accordion-expanded-content--IsExpanded}} pf-m-expanded{{/if}}{{#if accordion-expanded-content--IsFixed}} pf-m-fixed{{/if}}{{#if accordion-expanded-content--modifier}} {{accordion-expanded-content--modifier}}{{/if}}"
   {{#unless accordion-expanded-content--IsExpanded}}hidden{{/unless}}
+  {{#if accordion-expanded-content--IsFixed}}
+    {{#unless accordion--IsDefinitionList}}
+      role="region"
```

Comment:
@thatblindgeye according to the docs you linked:

> Additionally, when adding tabIndex in this scenario, add role='region' and an aria-label that describes the scrollable element or an aria-labelledby pointing to a heading with a descriptive, accessible name.

Is it true that the region/label should also only be present when there is a visible scrollbar? If so, I think we need to move that into the conditional that checks for `accordion-expanded-content--HasNoScrollbar`.

Reply:
I think that would make sense. In React for the Page component as an example, you *could* pass in `hasOverflowScroll` which applies a tabindex even when there is no scrollbar, but ideally you'd only want to include it when there is an actual scrollbar.